hist/hist/src/TH3.cxx

```diff
@@ -1084,8 +1092,12 @@ Double_t TH3::GetCovariance(Int_t axis1, Int_t axis2) const
 ////////////////////////////////////////////////////////////////////////////////
 /// Return 3 random numbers along axis x , y and z distributed according
 /// the cell-contents of a 3-dim histogram
+/// @param x  reference to random generated x value
+/// @param y  reference to random generated y value
+/// @param z  reference to random generated z value
+/// @param rng (optional) Random number generator pointer used (default is gRandom)
```

Comment:
```suggestion
/// @param[out] x  reference to random generated x value
/// @param[out] y  reference to random generated y value
/// @param[out] z  reference to random generated z value
/// @param[in] rng (optional) Random number generator pointer used (default is gRandom)
```
Can use optional doxygen tag to show what's an "in" and what's an "out" parameter.